*** 99-platformio-udev.rules	2020-12-23 17:10:15.769924225 +0100
--- 99-platformio-udev.rules.new	2020-12-23 18:22:33.358156770 +0100
*************** ATTRS{idVendor}=="1cbe", ATTRS{idProduct
*** 67,75 ****
  #TI MSP430 Launchpad
  ATTRS{idVendor}=="0451", ATTRS{idProduct}=="f432", MODE="0666", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_PORT_IGNORE}="1"
  
! #GD32V DFU Bootloader
  ATTRS{idVendor}=="28e9", ATTRS{idProduct}=="0189", MODE="0666", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_PORT_IGNORE}="1"
  
  #
  # Debuggers
  #
--- 67,78 ----
  #TI MSP430 Launchpad
  ATTRS{idVendor}=="0451", ATTRS{idProduct}=="f432", MODE="0666", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_PORT_IGNORE}="1"
  
! #GD32V DFU Bootloader and RV-Link JTAG key
  ATTRS{idVendor}=="28e9", ATTRS{idProduct}=="0189", MODE="0666", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_PORT_IGNORE}="1"
  
+ #GD32V RV-Link VCOM serial port forwarding of target-UART0
+ ATTRS{idVendor}=="28e9", ATTRS{idProduct}=="018a", MODE="0666", ENV{ID_MM_DEVICE_IGNORE}="1", ENV{ID_MM_PORT_IGNORE}="1"
+ 
  #
  # Debuggers
  #
